src/makeRobot.js

```diff
@@ -39,6 +39,74 @@
  */
 function makeRobot(name, wheels, version) {
   // write code here
+  const robot = {
+    name,
+    wheels,
+    version,
+    coords: {
+      x: 0,
+      y: 0,
+    },
+
+    get info() {
+      const info = `name: ${this.name}, `
+      + `chip version: ${this.version}, `
+      + `wheels: ${this.wheels}`;
+
+      return info;
+    },
+
+    get location() {
+      return `${this.name}: x=${this.coords.x}, y=${this.coords.y}`;
+    },
+
+    goForward(value = 1) {
+      if (value < 0) {
+        return robot;
+      }
+
+      this.coords.y += value;
+
+      return robot;
```

Comment:
simplify just like this and apply for other cases as well
```suggestion
      if (value > 0) {
        this.coords.y += value;
      }

      return this;
```